src/lib/viewers/controls/rotate/RotateControl.scss

```diff
@@ -0,0 +1,19 @@
+@import '../styles';
+
+.bp-RotateControl {
+    @include bp-ControlGroup;
+}
+
+.bp-RotateControl-button {
+    @include bp-ControlButton($width: 32px);
+}
+
+.bp-RotateControl-current {
```

Comment:
Is this used or was this copy/pasted from elsewhere?

Reply:
This was copied from ZoomControls, will remove!